easybuild/tools/build_log.py

```diff
@@ -186,10 +186,10 @@ def exception(self, msg, *args):
 _init_easybuildlog = fancylogger.getLogger(fname=False)
 
 
-def init_logging(logfile, logtostdout=False, silent=False):
+def init_logging(logfile, logtostdout=False, silent=False, color=None):
```

Comment:
(Kenneth Hoste, Fri, Mar 25, 2016 at 03:44:54PM -0700:)
> > @@ -186,10 +186,10 @@ def exception(self, msg, *args):
> >  _init_easybuildlog = fancylogger.getLogger(fname=False)
> >
> >
> > -def init_logging(logfile, logtostdout=False, silent=False):
> > +def init_logging(logfile, logtostdout=False, silent=False, color=None):
>
> the default for `color` here should be a string value?

Yep, changed default to `auto`